### PR TITLE
Resolves #177 Adjust build.xml to correcty call through to VI specific config.vi.xml.

### LINKED ISSUE #177: Update old-tck build.xml config.vi enable.jaspic and disable.jaspic targets to call through the VI build only.
The root build.xml should call the vi build.xml for these targets, the vi build.xml then in turn can choose to call s1as.xml if it makes sense.

### PR DESCRIPTION
**Pull request by darranl**
The pattern of updates here is to match the relationship from the platform level TCK.

### CODE REVIEW

File: tck/old-tck/source/install/jaspic/bin/build.xml [hunk 1]
```diff
@@ -51,7 +51,7 @@
     <target name="runclient" 
             depends="init, set.keywords, prepare.work.report.dirs, check.selected.test, 
                      no.selected.test, check.java.options, no.java.options, translatepath">
-        <ant antfile="config.vi.xml" dir="${bin.dir}/xml/impl/${impl.vi}" target="config.acc.template"/>
+        <ant antfile="config.vi.xml" dir="${bin.dir}/xml/impl/${impl.ri}" target="config.acc.template"/>
```
Comment (scottmarlow) [inline tck/old-tck/source/install/jaspic/bin/build.xml:54]: Is GlassFish server used for testing other implementations?

File: tck/old-tck/source/install/jaspic/bin/ts.jte [hunk 2]
```diff
@@ -520,6 +520,7 @@ impl.vi=glassfish
 impl.vi.deploy.dir=${s1as.domain}/autodeploy
 impl.deploy.timeout.multiplier=1200
 
+impl.ri=glassfish
```
Comment (scottmarlow) [inline tck/old-tck/source/install/jaspic/bin/ts.jte:523]: We don't consider GlassFish a reference implementation but historically we do of course.  We still have a lot of various references to `ri` things (in TCKs) but now they are just referencing any compatible implementation.

**Comment by arjantijms** [comment]
We should probably avoid the terms "ri" and "jaspic" for new code (including comments).

File: tck/old-tck/source/install/jaspic/bin/ts.jte (hunk 2 above)
Comment (darranl) [reply tck/old-tck/source/install/jaspic/bin/ts.jte:523]: @scottmarlow the changes in this PR are only about following the same structure as is used in the platform TCK to allow alignment of porting kits, the platform TCK uses RI so this matches the platform TCK for this specific ant target.

Comment (scottmarlow) [reply tck/old-tck/source/install/jaspic/bin/ts.jte:523]: Thanks for explaining Darran.  I'm fine with it for this pull request which introduces more compatibility for implementations that have previously run against the Platform TCK Authentication tests.

Going forward, if the old Authentication TCK tests will be refactored into the new Authentication TCK, that might be a better time to drop compatibility with the Platform TCK (legacy way).

**Review by scottmarlow — APPROVED** [review]

**Review by arjantijms — APPROVED** [review]

**Pull request merged**